Chisel/Makefile

```diff
@@ -0,0 +1,14 @@
+PREFIX ?= /usr/local/lib
+TEMPORARY_FOLDER ?= /tmp/Chisel.dst
+
+install:
+	xcodebuild \
+		-scheme Chisel \
+		-configuration Release \
+		-sdk iphonesimulator \
+		install \
+		DSTROOT="$(TEMPORARY_FOLDER)" \
+		INSTALL_PATH=/ \
```

Comment:
I don't know much about Xcode's install action. Why install into a temp path and then copy from there into the true destination?

Reply:
Ah yes, nice I didn't re-think this after I made some other changes, updated